lambda/translmod.ml

```diff
@@ -34,8 +34,13 @@ type unsafe_component =
   | Unsafe_typext
 
 type unsafe_info =
-  | Unsafe of { reason:unsafe_component; loc:Location.t; subid:Ident.t }
+  | Unsafe of {
+    reason:unsafe_component;
+    loc:Location.t;
+    path: Path.t
+    }
```

Comment:
I would 2-indent the fields, so that they are more indented than the closing brace.